testing/sdk_networks.py

```diff
@@ -44,17 +46,27 @@ def check_task_network(task_name, expected_network_name="dcos"):
                     )
 
 
+@retrying.retry(wait_fixed=1000, stop_max_delay=5 * 1000, retry_on_result=lambda res: not res)
+def wait_for_endpoint_info(
```

Comment:
My thought here is as follows: Consider splitting this into two functions:
* `wait_for_endpoint_json()`:
** returns a `typing.Dict`
** Could be left as `wait_for_endpoint_info`
* `wait_for_endpoint_string()`:
** returns a `str`
** returns `output.strip()` since we are capturing stdout, and this adds additional whitespace.

This makes the distinction at the call-site more explicit.

What are your thoughts on this?

Reply:
Definitely sounds reasonable. I'll also try then to clean up the unnecessary used `strip()` calls which are used currently.